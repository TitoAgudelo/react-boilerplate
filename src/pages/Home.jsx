import React from 'react'

import Header from '../components/Header/Header';
import Posts from '../components/Posts/Posts';


function Home() {
  return (
    <>
      <Header />
      <Posts />
    </>
  );
}

export default Home;
