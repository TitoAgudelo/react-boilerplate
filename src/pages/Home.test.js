import React from 'react';
import { shallow } from 'enzyme';
import Home from './Home';

let wrapped = shallow(<Home />);

describe('Home', () => {
  it('should render the Page Component as snapshot', () => {   
    expect(wrapped).toMatchSnapshot();
  });
});
