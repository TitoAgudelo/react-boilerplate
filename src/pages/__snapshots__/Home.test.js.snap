// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render the Title Component correctly 1`] = `
<Fragment>
  <Header />
  <Posts />
</Fragment>
`;

exports[`Home should render the Page Component as snapshot 1`] = `
<Fragment>
  <Header />
  <Posts />
</Fragment>
`;
